src/main/target/common_post.h

```diff
@@ -267,6 +267,10 @@
 #undef USE_CMS_FAILSAFE_MENU
 #endif
 
+#ifndef USE_CMS_FAILSAFE_MENU
+#undef USE_CMS_GPS_RESCUE_MENU
```

Comment:
This should also be undefined in here if `USE_GPS_RESCUE` isn't defined.

Reply:
will do 👍 .. lack of sleep and caffeine  :)